AltStore/LaunchViewController.swift

```diff
@@ -42,14 +42,21 @@ class LaunchViewController: RSTLaunchViewController
             self.destinationViewController = self.storyboard!.instantiateViewController(withIdentifier: "tabBarController") as! TabBarController
         }
         super.viewDidLoad()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(true)
         start_em_proxy(bind_addr: Consts.Proxy.serverURL)
         
         guard let pf = fetchPairingFile() else {
-            displayError("ALTPairingFile not found.")
+            displayError("Device pairing file not found.")
             return
         }
         set_usbmuxd_socket()
-        start_minimuxer(pairing_file: pf)
+        let res = start_minimuxer(pairing_file: pf)
+        if res != 0 {
+            displayError("minimuxer failed to start. Incorrect arguments were passed.")
```

Comment:
Incorrect arguments is the only error this part of the rust can throw

Reply:
But what exactly does "incorrect arguments" *mean*?

Reply:
Is this a condition that can only be triggered by a bug in the code? (And not some error on the user's part).
If so, this is fine.

Reply:
google.

Reply:
??

Reply:
it's an error message, we dont need to make a whole chat about it.

PRs need to be quick and simple please

Reply:
Sheesh, I was only a little confused.
Reading the rust code, I wasn't sure if an error loading the pairing file could cause the error. If so, it could be the user's fault by supplying a bad pairing file, and we should give them a nicer error that tells them so.
If not, then fine.